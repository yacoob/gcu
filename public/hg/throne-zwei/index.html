<html lang='en'>
<head>
  <meta charset='utf-8'>
  <meta http-equiv='X-UA-Compatible' content='IE=edge'>
  <meta name='viewport' content='width=device-width, initial-scale=1'>
  <!--[if lt IE 9]>
  <script src='https://oss.maxcdn.com/html5shiv/3.7.2/html5shiv.min.js'></script>
  <script src='https://oss.maxcdn.com/respond/1.4.2/respond.min.js'></script>
  <![endif]-->
  <link href='https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/3.3.7/css/bootstrap.min.css' rel='stylesheet'>
  <link href='/css/gcu.css' rel='stylesheet'>
  <link href='/i/favicon.ico' rel='icon' type='image/x-icon'/>
  <link href='//syn.tactical-grace.net/f/full/gcu-00441.jpg' rel='image_src'/>
  <meta property='og:image' content='https://syn.tactical-grace.net/f/full/gcu-00441.jpg'/>
  <meta name='description' content='tactical grace presents: Throne Zwei'/>
  <link href='http://gcu.tactical-grace.net/hg/throne-zwei/' rel='canonical'/>
  <link rel='alternate' type='application/rss+xml' title='tactical grace feed' href='/index.xml'>
  <title>HG: Throne Zwei | tactical grace</title>
  <meta content='HG: Throne Zwei | tactical grace' itemprop='name'/>
</head>
<body>
<div class='container'>
<div class='row'><!-- title and mininav -->
  <div class='col-sm-12 col-md-8 text-right'>
  <h1 class='title'><a href='/hg/throne-zwei/'>HG: Throne Zwei</a></h1>
</div> <!-- column -->
  <div class='col-sm-12 col-md-4 text-right voffset-top'>
    <a data-toggle='tooltip' title='main page' href='/'>
    <button class='btn btn-default btn-sm' type='button'><span class='glyphicon glyphicon-home'></span></button></a>
    <a data-toggle='tooltip' title='contact the author' href='https://goo.gl/forms/iJEsfLjBy1FE86aQ2'>
    <button class='btn btn-default btn-sm' type='button'><span class='glyphicon glyphicon-envelope'></span></button></a>
    <a data-toggle='tooltip' title='grade page' href='/hg/'>
    <button class='btn btn-default btn-sm' type='button'><span class='glyphicon glyphicon-th'></span></button></a>
    <a data-toggle='tooltip' title='previous kit' href='/hg/stark-jegan/'>
    <button class='btn btn-default btn-sm' type='button'><span class='glyphicon glyphicon-chevron-left'></span></button></a>
    <a data-toggle='tooltip' title='next kit' href='/hg/turn-a/'>
    <button class='btn btn-default btn-sm' type='button'><span class='glyphicon glyphicon-chevron-right'></span></button></a>
  </div>
</div> <!-- row -->
  <div class='row voffset-top'>
    <h2 id='2014-03-09'>2014-03-09</h2>
  </div>
<div class='row voffset-small'>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='//syn.tactical-grace.net/f/full/gcu-00431.jpg'>
      <img class='img-rounded img-responsive' src='//syn.tactical-grace.net/f/thumb/gcu-00431.jpg' title='Primer sticking out from underneath the paint.' alt='Primer sticking out from underneath the paint.' />
      <span class='glyphicon glyphicon-fullscreen gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='//syn.tactical-grace.net/f/full/gcu-00430.jpg'>
      <img class='img-rounded img-responsive' src='//syn.tactical-grace.net/f/thumb/gcu-00430.jpg' title='Happy with this one.' alt='Happy with this one.' />
      <span class='glyphicon glyphicon-fullscreen gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='//syn.tactical-grace.net/f/full/gcu-00432.jpg'>
      <img class='img-rounded img-responsive' src='//syn.tactical-grace.net/f/thumb/gcu-00432.jpg' title='Easy to overdose on the paint on curved surfaces.' alt='Easy to overdose on the paint on curved surfaces.' />
      <span class='glyphicon glyphicon-fullscreen gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='//syn.tactical-grace.net/f/full/gcu-00434.jpg'>
      <img class='img-rounded img-responsive' src='//syn.tactical-grace.net/f/thumb/gcu-00434.jpg' title='Primed today; will be white, yellow and silver.' alt='Primed today; will be white, yellow and silver.' />
      <span class='glyphicon glyphicon-fullscreen gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='//syn.tactical-grace.net/f/full/gcu-00429.jpg'>
      <img class='img-rounded img-responsive' src='//syn.tactical-grace.net/f/thumb/gcu-00429.jpg' title='Painted today; red and black.' alt='Painted today; red and black.' />
      <span class='glyphicon glyphicon-fullscreen gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='//syn.tactical-grace.net/f/full/gcu-00433.jpg'>
      <img class='img-rounded img-responsive' src='//syn.tactical-grace.net/f/thumb/gcu-00433.jpg' title='Primed some time ago; will be gunmetal.' alt='Primed some time ago; will be gunmetal.' />
      <span class='glyphicon glyphicon-fullscreen gallery-icon'></span>
    </a>
  </div>
</div>
</div>
  <div class='row voffset-top'>
    <h2 id='2014-03-19'>2014-03-19</h2>
  </div>
<div class='row voffset-small'>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='//syn.tactical-grace.net/f/full/gcu-00440.jpg'>
      <img class='img-rounded img-responsive' src='//syn.tactical-grace.net/f/thumb/gcu-00440.jpg' title='Black is the new, well, black.' alt='Black is the new, well, black.' />
      <span class='glyphicon glyphicon-fullscreen gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='//syn.tactical-grace.net/f/full/gcu-00441.jpg'>
      <img class='img-rounded img-responsive' src='//syn.tactical-grace.net/f/thumb/gcu-00441.jpg' />
      <span class='glyphicon glyphicon-fullscreen gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='//syn.tactical-grace.net/f/full/gcu-00039.jpg'>
      <img class='img-rounded img-responsive' src='//syn.tactical-grace.net/f/thumb/gcu-00039.jpg' title='Original colours for all three Thrones.' alt='Original colours for all three Thrones.' />
      <span class='glyphicon glyphicon-fullscreen gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='//syn.tactical-grace.net/f/full/gcu-00435.jpg'>
      <img class='img-rounded img-responsive' src='//syn.tactical-grace.net/f/thumb/gcu-00435.jpg' title='Cleaning the airbrush.' alt='Cleaning the airbrush.' />
      <span class='glyphicon glyphicon-fullscreen gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='//syn.tactical-grace.net/f/full/gcu-00436.jpg'>
      <img class='img-rounded img-responsive' src='//syn.tactical-grace.net/f/thumb/gcu-00436.jpg' title='Silver sword edge.' alt='Silver sword edge.' />
      <span class='glyphicon glyphicon-fullscreen gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='//syn.tactical-grace.net/f/full/gcu-00437.jpg'>
      <img class='img-rounded img-responsive' src='//syn.tactical-grace.net/f/thumb/gcu-00437.jpg' title='Yellow vents.' alt='Yellow vents.' />
      <span class='glyphicon glyphicon-fullscreen gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='//syn.tactical-grace.net/f/full/gcu-00439.jpg'>
      <img class='img-rounded img-responsive' src='//syn.tactical-grace.net/f/thumb/gcu-00439.jpg' title='Hard to check coverage with white over white over white.' alt='Hard to check coverage with white over white over white.' />
      <span class='glyphicon glyphicon-fullscreen gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='//syn.tactical-grace.net/f/full/gcu-00438.jpg'>
      <img class='img-rounded img-responsive' src='//syn.tactical-grace.net/f/thumb/gcu-00438.jpg' title='Gunmetal for the gun. Obviously.' alt='Gunmetal for the gun. Obviously.' />
      <span class='glyphicon glyphicon-fullscreen gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='//syn.tactical-grace.net/f/full/gcu-00442.jpg'>
      <img class='img-rounded img-responsive' src='//syn.tactical-grace.net/f/thumb/gcu-00442.jpg' title='Left: varnished. Right: unvarnished.' alt='Left: varnished. Right: unvarnished.' />
      <span class='glyphicon glyphicon-fullscreen gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='//syn.tactical-grace.net/f/full/gcu-00443.jpg'>
      <img class='img-rounded img-responsive' src='//syn.tactical-grace.net/f/thumb/gcu-00443.jpg' title='Left: unvarnished. Right: varnished.' alt='Left: unvarnished. Right: varnished.' />
      <span class='glyphicon glyphicon-fullscreen gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='//syn.tactical-grace.net/f/full/gcu-00444.jpg'>
      <img class='img-rounded img-responsive' src='//syn.tactical-grace.net/f/thumb/gcu-00444.jpg' title='Left: unvarnished. Right: varnished, badly.' alt='Left: unvarnished. Right: varnished, badly.' />
      <span class='glyphicon glyphicon-fullscreen gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='//syn.tactical-grace.net/f/full/gcu-00445.jpg'>
      <img class='img-rounded img-responsive' src='//syn.tactical-grace.net/f/thumb/gcu-00445.jpg' title='Bottom: unvarnished. Top: varnished.' alt='Bottom: unvarnished. Top: varnished.' />
      <span class='glyphicon glyphicon-fullscreen gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='//syn.tactical-grace.net/f/full/gcu-00446.jpg'>
      <img class='img-rounded img-responsive' src='//syn.tactical-grace.net/f/thumb/gcu-00446.jpg' title='Lighter fluid ate through everything.' alt='Lighter fluid ate through everything.' />
      <span class='glyphicon glyphicon-fullscreen gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='//syn.tactical-grace.net/f/full/gcu-00447.jpg'>
      <img class='img-rounded img-responsive' src='//syn.tactical-grace.net/f/thumb/gcu-00447.jpg' title='Left: unvarnished. Right: varnished.' alt='Left: unvarnished. Right: varnished.' />
      <span class='glyphicon glyphicon-fullscreen gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='//syn.tactical-grace.net/f/full/gcu-00448.jpg'>
      <img class='img-rounded img-responsive' src='//syn.tactical-grace.net/f/thumb/gcu-00448.jpg' title='Left: unvarnished. Right: varnished.' alt='Left: unvarnished. Right: varnished.' />
      <span class='glyphicon glyphicon-fullscreen gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='//syn.tactical-grace.net/f/full/gcu-00449.jpg'>
      <img class='img-rounded img-responsive' src='//syn.tactical-grace.net/f/thumb/gcu-00449.jpg' title='Rest in pieces.' alt='Rest in pieces.' />
      <span class='glyphicon glyphicon-fullscreen gallery-icon'></span>
    </a>
  </div>
</div>
</div>
<div class='row'>
  <div class='col-sm-12 col-md-7 col-md-offset-1 voffset-top-small'>
    <div class='btn-group btn-group-justified btn-group-lg' role='group'>
      <a class='btn btn-default' href='/hg/' role='button'>
        <span class='glyphicon glyphicon-pawn hidden-xs'></span>
        HG</a>
      <a class='btn btn-default' href='/rg/' role='button'>
        <span class='glyphicon glyphicon-knight hidden-xs'></span>
        RG</a>
      <a class='btn btn-default' href='/mg/' role='button'>
        <span class='glyphicon glyphicon-bishop hidden-xs'></span>
        MG</a>
      <a class='btn btn-default' href='/pg/' role='button'>
        <span class='glyphicon glyphicon-king hidden-xs'></span>
        PG</a>
      <a class='btn btn-default' href='/varia/' role='button'>
        <span class='glyphicon glyphicon-tower hidden-xs'></span>
        Varia</a>
    </div> <!-- btn-group -->
  </div> <!-- column -->
  <div class='col-sm-12 col-md-4 voffset-top-small'>
    <div class='btn-group btn-group-justified btn-group-lg' role='group'>
      <a class='btn btn-danger btn-lg' href='/everything/' role='button'>
        <span class='glyphicon glyphicon-fire'></span>
        Show me everything!</a>
    </div>
  </div> <!-- column -->
</div> <!-- row -->
</div> <!-- container -->
<link href='http://fonts.googleapis.com/css?family=Exo' rel='stylesheet' type='text/css'>
<script> <!-- analytics -->
  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
  })(window,document,'script','//www.google-analytics.com/analytics.js','ga');

  ga('create', 'UA-35396418-1', 'auto');
  ga('send', 'pageview');
</script>
<link type='text/css' rel='stylesheet' href='https://cdnjs.cloudflare.com/ajax/libs/lightgallery/1.6.4/css/lightgallery.min.css' />
<!-- include lightbox js/css here -->
<script src='https://cdnjs.cloudflare.com/ajax/libs/jquery/2.2.0/jquery.min.js'></script>
<script src='https://cdnjs.cloudflare.com/ajax/libs/lightgallery/1.6.4/js/lightgallery.min.js'></script>
<script src='https://cdnjs.cloudflare.com/ajax/libs/lg-fullscreen/1.0.1/lg-fullscreen.min.js'></script>
<script src='https://cdnjs.cloudflare.com/ajax/libs/lg-thumbnail/1.1.0/lg-thumbnail.min.js'></script>
<script src='https://cdnjs.cloudflare.com/ajax/libs/lg-autoplay/1.0.4/lg-autoplay.min.js'></script>
<script src='https://cdnjs.cloudflare.com/ajax/libs/lg-zoom/1.1.0/lg-zoom.min.js'></script>
<script src='https://cdnjs.cloudflare.com/ajax/libs/lg-video/1.1.0/lg-video.min.js'></script>
<script src='/js/gcu.js'></script>
<script type='text/javascript'>
$(document).ready(gcu.postPageHandler);
</script>
</body></html>