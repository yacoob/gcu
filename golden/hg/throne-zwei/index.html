<html lang='en'>
<head>
  <meta charset='utf-8'>
  <meta http-equiv='X-UA-Compatible' content='IE=edge'>
  <meta name='viewport' content='width=device-width, initial-scale=1'>
  <link href='https://cdn.jsdelivr.net/npm/bootstrap3@3/dist/css/bootstrap.min.css' rel='stylesheet'>
  <link href='/css/gcu.css' rel='stylesheet'>
  <link href='/i/favicon.ico' rel='icon' type='image/x-icon'/>
  <link href='/photos/full/gcu-00441.jpg' rel='image_src'/>
  <meta property='og:image' content='https://gcu.tactical-grace.net/photos/full/gcu-00441.jpg'/>
  <meta name='twitter:card' content='summary_large_image'/>
  <meta name='twitter:image' content='https://gcu.tactical-grace.net/photos/full/gcu-00441.jpg'/>
  <meta name='description' content='tactical grace presents: Throne Zwei'/>
  <meta name='twitter:title' content='tactical grace presents: Throne Zwei'/>
  <link href='https://gcu.tactical-grace.net/hg/throne-zwei/' rel='canonical'/>
  <link rel='alternate' type='application/rss+xml' title='tactical grace feed' href='/index.xml'>
  <title>HG: Throne Zwei | tactical grace</title>
  <meta content='HG: Throne Zwei | tactical grace' itemprop='name'/>
</head>
<body>
<div class='container'>
<div class='row'><!-- title and mininav -->
  <div class='col-sm-12 col-md-8 text-right'>
  <h1 class='title'><a href='/hg/throne-zwei/'>HG: Throne Zwei</a></h1>
</div> <!-- column -->
  <div class='col-sm-12 col-md-4 text-right voffset-top'>
    <a data-toggle='tooltip' title='main page' href='/'>
    <button class='btn btn-default btn-sm' type='button'><span class='glyphicon glyphicon-home'></span></button></a>
    <a data-toggle='tooltip' title='contact the author' href='https://goo.gl/forms/iJEsfLjBy1FE86aQ2'>
    <button class='btn btn-default btn-sm' type='button'><span class='glyphicon glyphicon-envelope'></span></button></a>
    <a data-toggle='tooltip' title='grade page' href='/hg/'>
    <button class='btn btn-default btn-sm' type='button'><span class='glyphicon glyphicon-th'></span></button></a>
    <a data-toggle='tooltip' title='previous kit' href='/hg/stark-jegan/'>
    <button class='btn btn-default btn-sm' type='button'><span class='glyphicon glyphicon-chevron-left'></span></button></a>
    <a data-toggle='tooltip' title='next kit' href='/hg/turn-a/'>
    <button class='btn btn-default btn-sm' type='button'><span class='glyphicon glyphicon-chevron-right'></span></button></a>
  </div>
</div> <!-- row -->
  <div class='row voffset-top'>
    <h2 id='2014-03-09'>2014-03-09</h2>
  </div>
<div class='row voffset-small'>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='/photos/full/gcu-00431.jpg' data-sub-html='Primer sticking out from underneath the paint.'>
      <img class='img-rounded img-responsive' src='/photos/thumb/gcu-00431.jpg' alt='Primer sticking out from underneath the paint.' />
      <span class='glyphicon  gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='/photos/full/gcu-00430.jpg' data-sub-html='Happy with this one.'>
      <img class='img-rounded img-responsive' src='/photos/thumb/gcu-00430.jpg' alt='Happy with this one.' />
      <span class='glyphicon  gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='/photos/full/gcu-00432.jpg' data-sub-html='Easy to overdose on the paint on curved surfaces.'>
      <img class='img-rounded img-responsive' src='/photos/thumb/gcu-00432.jpg' alt='Easy to overdose on the paint on curved surfaces.' />
      <span class='glyphicon  gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='/photos/full/gcu-00434.jpg' data-sub-html='Primed today; will be white, yellow and silver.'>
      <img class='img-rounded img-responsive' src='/photos/thumb/gcu-00434.jpg' alt='Primed today; will be white, yellow and silver.' />
      <span class='glyphicon  gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='/photos/full/gcu-00429.jpg' data-sub-html='Painted today; red and black.'>
      <img class='img-rounded img-responsive' src='/photos/thumb/gcu-00429.jpg' alt='Painted today; red and black.' />
      <span class='glyphicon  gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='/photos/full/gcu-00433.jpg' data-sub-html='Primed some time ago; will be gunmetal.'>
      <img class='img-rounded img-responsive' src='/photos/thumb/gcu-00433.jpg' alt='Primed some time ago; will be gunmetal.' />
      <span class='glyphicon  gallery-icon'></span>
    </a>
  </div>
</div>
</div>
  <div class='row voffset-top'>
    <h2 id='2014-03-19'>2014-03-19</h2>
  </div>
<div class='row voffset-small'>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='/photos/full/gcu-00440.jpg' data-sub-html='Black is the new, well, black.'>
      <img class='img-rounded img-responsive' src='/photos/thumb/gcu-00440.jpg' alt='Black is the new, well, black.' />
      <span class='glyphicon  gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='/photos/full/gcu-00441.jpg'>
      <img class='img-rounded img-responsive' src='/photos/thumb/gcu-00441.jpg' />
      <span class='glyphicon  gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='/photos/full/gcu-00039.jpg' data-sub-html='Original colours for all three Thrones.'>
      <img class='img-rounded img-responsive' src='/photos/thumb/gcu-00039.jpg' alt='Original colours for all three Thrones.' />
      <span class='glyphicon  gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='/photos/full/gcu-00435.jpg' data-sub-html='Cleaning the airbrush.'>
      <img class='img-rounded img-responsive' src='/photos/thumb/gcu-00435.jpg' alt='Cleaning the airbrush.' />
      <span class='glyphicon  gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='/photos/full/gcu-00436.jpg' data-sub-html='Silver sword edge.'>
      <img class='img-rounded img-responsive' src='/photos/thumb/gcu-00436.jpg' alt='Silver sword edge.' />
      <span class='glyphicon  gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='/photos/full/gcu-00437.jpg' data-sub-html='Yellow vents.'>
      <img class='img-rounded img-responsive' src='/photos/thumb/gcu-00437.jpg' alt='Yellow vents.' />
      <span class='glyphicon  gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='/photos/full/gcu-00439.jpg' data-sub-html='Hard to check coverage with white over white over white.'>
      <img class='img-rounded img-responsive' src='/photos/thumb/gcu-00439.jpg' alt='Hard to check coverage with white over white over white.' />
      <span class='glyphicon  gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='/photos/full/gcu-00438.jpg' data-sub-html='Gunmetal for the gun. Obviously.'>
      <img class='img-rounded img-responsive' src='/photos/thumb/gcu-00438.jpg' alt='Gunmetal for the gun. Obviously.' />
      <span class='glyphicon  gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='/photos/full/gcu-00442.jpg' data-sub-html='Left: varnished. Right: unvarnished.'>
      <img class='img-rounded img-responsive' src='/photos/thumb/gcu-00442.jpg' alt='Left: varnished. Right: unvarnished.' />
      <span class='glyphicon  gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='/photos/full/gcu-00443.jpg' data-sub-html='Left: unvarnished. Right: varnished.'>
      <img class='img-rounded img-responsive' src='/photos/thumb/gcu-00443.jpg' alt='Left: unvarnished. Right: varnished.' />
      <span class='glyphicon  gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='/photos/full/gcu-00444.jpg' data-sub-html='Left: unvarnished. Right: varnished, badly.'>
      <img class='img-rounded img-responsive' src='/photos/thumb/gcu-00444.jpg' alt='Left: unvarnished. Right: varnished, badly.' />
      <span class='glyphicon  gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='/photos/full/gcu-00445.jpg' data-sub-html='Bottom: unvarnished. Top: varnished.'>
      <img class='img-rounded img-responsive' src='/photos/thumb/gcu-00445.jpg' alt='Bottom: unvarnished. Top: varnished.' />
      <span class='glyphicon  gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='/photos/full/gcu-00446.jpg' data-sub-html='Lighter fluid ate through everything.'>
      <img class='img-rounded img-responsive' src='/photos/thumb/gcu-00446.jpg' alt='Lighter fluid ate through everything.' />
      <span class='glyphicon  gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='/photos/full/gcu-00447.jpg' data-sub-html='Left: unvarnished. Right: varnished.'>
      <img class='img-rounded img-responsive' src='/photos/thumb/gcu-00447.jpg' alt='Left: unvarnished. Right: varnished.' />
      <span class='glyphicon  gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='/photos/full/gcu-00448.jpg' data-sub-html='Left: unvarnished. Right: varnished.'>
      <img class='img-rounded img-responsive' src='/photos/thumb/gcu-00448.jpg' alt='Left: unvarnished. Right: varnished.' />
      <span class='glyphicon  gallery-icon'></span>
    </a>
  </div>
</div>
<div class='col-xs-6 col-md-3 col-lg-2'>
  <div class='gallery-entry voffset-bottom-small'>
    <a class='gallery' href='/photos/full/gcu-00449.jpg' data-sub-html='Rest in pieces.'>
      <img class='img-rounded img-responsive' src='/photos/thumb/gcu-00449.jpg' alt='Rest in pieces.' />
      <span class='glyphicon  gallery-icon'></span>
    </a>
  </div>
</div>
</div>
<div class='row'>
  <div class='col-sm-12 col-md-7 col-md-offset-1 voffset-top-small'>
    <div class='btn-group btn-group-justified btn-group-lg' role='group'>
      <a class='btn btn-default' href='/hg/' role='button'>
        <span class='glyphicon glyphicon-pawn hidden-xs'></span>
        HG</a>
      <a class='btn btn-default' href='/rg/' role='button'>
        <span class='glyphicon glyphicon-knight hidden-xs'></span>
        RG</a>
      <a class='btn btn-default' href='/mg/' role='button'>
        <span class='glyphicon glyphicon-bishop hidden-xs'></span>
        MG</a>
      <a class='btn btn-default' href='/pg/' role='button'>
        <span class='glyphicon glyphicon-king hidden-xs'></span>
        PG</a>
      <a class='btn btn-default' href='/varia/' role='button'>
        <span class='glyphicon glyphicon-tower hidden-xs'></span>
        Varia</a>
    </div> <!-- btn-group -->
  </div> <!-- column -->
  <div class='col-sm-12 col-md-4 voffset-top-small'>
    <div class='btn-group btn-group-justified btn-group-lg' role='group'>
      <a class='btn btn-danger btn-lg' href='/everything/' role='button'>
        <span class='glyphicon glyphicon-fire'></span>
        Show me everything!</a>
    </div>
  </div> <!-- column -->
</div> <!-- row -->
</div> <!-- container -->
<link href='https://fonts.googleapis.com/css?family=Exo' rel='stylesheet' type='text/css'>
<script> <!-- analytics -->
  (function(i,s,o,g,r,a,m){i['GoogleAnalyticsObject']=r;i[r]=i[r]||function(){
  (i[r].q=i[r].q||[]).push(arguments)},i[r].l=1*new Date();a=s.createElement(o),
  m=s.getElementsByTagName(o)[0];a.async=1;a.src=g;m.parentNode.insertBefore(a,m)
  })(window,document,'script','//www.google-analytics.com/analytics.js','ga');

  ga('create', 'UA-35396418-1', 'auto');
  ga('send', 'pageview');
</script>
<link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/lightgallery.js@1/dist/css/lightgallery.min.css" />
<script src="https://cdn.jsdelivr.net/npm/lightgallery.js@1/dist/js/lightgallery.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/lg-thumbnail.js@1/dist/lg-thumbnail.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/lg-zoom.js@1/dist/lg-zoom.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/lg-video.js@1/dist/lg-video.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/lg-fullscreen.js@1/dist/lg-fullscreen.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/lg-autoplay.js@1/dist/lg-autoplay.min.js"></script>
<script src='/js/gcu.js'></script>
<script type='text/javascript'>
document.addEventListener('DOMContentLoaded', gcu.postPageHandler);
</script>
</body></html>